xtask/src/codegen/gen_syntax.rs

```diff
@@ -247,14 +247,14 @@ fn generate_nodes(kinds: KindsSrc<'_>, grammar: &AstSrc) -> Result<String> {
         #(#display_impls)*
     };
 
-    let ast = ast.to_string().replace("T ! [ ", "T![").replace(" ] )", "])");
```

Comment:
CAn we keep `T ! [` -> `T![` transform?

Reply:
Sure thing. It just didn't seem to do anything.

Reply:
oh I see why...